src/styles/index.css

```diff
@@ -24,8 +24,28 @@ h1 {
   }
 }
 
-@screen xl {
-  h1 {
-    @apply text-3xl;
-  }
+/* Code block styles */
+.prose pre {
+  @apply mx-2 px-8 py-6 bg-gray-800 text-gray-200 rounded-md;
+}
+
+pre {
+  @apply mx-2 px-8 py-6 bg-gray-800 text-gray-200 rounded-md;
+}
+
+/* Contain typography to constrained width */
+.prose h1, h2, h3, h4, h5, p, ul, ol, li, span {
+  @apply max-w-2xl mx-auto;
 }
```

Comment:
**suggestion:** Each prose size modifier already comes with a [baked in max-width](https://github.com/tailwindlabs/tailwindcss-typography#overriding-max-width). Setting max-width on higher level, e.g. [in templates](https://github.com/eggheadio/egghead-next/blob/ma/prose-typography-styling/src/layouts/ultimate-guide.tsx#L34) would be my preferred way of doing it. More importantly, this change would disallow us from setting different max-width on all prose classes. If we want to modify prose, we should do so for each size modifier in tailwind config.

Reply:
Ah didn't realise the max-width changed between prose sizes. Can easily switch to modifying these in templates - will remove these.